### PR TITLE
Add support for EknServices version 3

### PR DESCRIPTION
**Pull request by ebassi**
https://phabricator.endlessm.com/T22061

### CODE REVIEW

**Comment by smspillaz** [comment]
Thanks!

**Comment by smspillaz** [comment]
Note to self: I'll need to update the EknServices3 service file in line with #2

File: com.endlessm.BaseEknServicesMultiplexer.json.in [hunk 1]
```diff
@@ -12,6 +12,10 @@
         "com.endlessm.EknServices2.Extension": {
             "directory": "build/eos-knowledge-services/2",
             "version": "@BRANCH@"
+        },
+        "com.endlessm.EknServices3.Extension": {
+            "directory": "build/eos-knowledge-services/3",
+            "version": "@BRANCH@"
```
[the remark this answers is not in the document]
Comment (ptomato) [reply com.endlessm.BaseEknServicesMultiplexer.json.in]: I looked in `jenkins-build-flatpak-manifest` and it expands to the last component of the Jenkins job name, or whatever is passed with `-b`. I'm thinking this should be `stable`, however ... @smspillaz?

[the remark this answers is not in the document]
Comment (smspillaz) [reply com.endlessm.BaseEknServicesMultiplexer.json.in]: Ah good point.

I think `@BRANCH@` is there for the the latest extension version only (and expands to either "master" or "stable" depending on the jenkins job). So you'll need to change the EknServices2  one to `2`

[the remark this answers is not in the document]
Comment (ptomato) [reply com.endlessm.BaseEknServicesMultiplexer.json.in]: OK, so this line stays as is but the EknServices2 one has to change. However, I suspect it needs to change to `stable`, not `2`?

**Pull request merged**